utils/gyb_syntax_support/ExprNodes.py

```diff
@@ -571,14 +571,14 @@
          ]),
 
     # SWIFT_ENABLE_TENSORFLOW
-    # Expression generalizing #gradient and #valueAndGradient.
-    # e.g. #gradient(foo(_:_:), wrt: .0, .1)
-    Node('GradientExpr', kind='Expr',
+    # e.g. "#gradient(foo(_:_:), wrt: .0, .1)"
+    Node('ReverseAutoDiffExpr', kind='Expr',
          traits=['Parenthesized'],
          children=[
              Child('Identifier', kind='Token',
                    token_choices=[
                        'PoundGradientToken',
+                       'ChainableGradientToken',
```

Comment:
Do you mean `PoundChainableGradientToken`?

Reply:
Aha. Good catch.